tests/unit/test_config.py

```diff
@@ -393,8 +450,15 @@ def test_dist_git_package_url():
 @pytest.mark.parametrize(
     "content",
     [
-        "---\nspecfile_path: packit.spec\nsynced_files:\n  - packit.spec\n  - .packit.yaml\n",
-        '{"specfile_path": "packit.spec", "synced_files": ["packit.spec", ".packit.yaml"]}',
+        "---\nspecfile_path: packit.spec\n"
+        "synced_files:\n"
+        "  - src: packit.spec\n"
+        "    dest: packit.spec\n"
+        "  - src: .packit.yaml\n"
+        "    dest: .packit.yaml\n",
+        '{"specfile_path": "packit.spec", "synced_files": '
+        '[{"src": "packit.spec", "dest": "packit.spec"}, '
+        '{"src": ".packit.yaml", "dest": ".packit.yaml"}]}',
```

Comment:
I'd keep the testing data simple. i.e. one synced file should be enough to verify the functionality.